include/bout/paralleltransform.hxx

```diff
@@ -125,7 +125,7 @@ public:
   }
 
   /// A 3D array, implemented as nested vectors
-  using arr3Dvec = std::vector<std::vector<std::vector<dcomplex>>>;
+  using arr3Dvec = std::vector<std::vector<Array<dcomplex>>>;
```

Comment:
Would #1505 help with this? What's the motivation for just changing the inner to an Array rather than the other nested levels?

Reply:
Yes, likely. I only *needed* the inner one to be an `Array`, and hadn't considered the others.